woocommerce/woocommerce-android
# [Better Errors] Redirect Login to Application Password Tutorial

Summary
==========
Fix issue #10803 by replacing the current error dialog with the WP Admin link to the Application Password Tutorial screen, which details the user on how to approach authentication through the web view better. 

Currently, the Application Password Tutorial screen cannot finish the authentication flow, so this behavior is behind a feature flag.

How to Test
==========
1. Make sure you have a store configured with something that can get in the way of the login, like the reCAPTCHA plugin.
2. Start the store login flow using the store address and direct user credentials.
3. Verify that the Username/Password screen redirects to the Application Password Tutorial screen.

Update release notes:
